backend/src/stats.js

```diff
@@ -80,6 +83,43 @@ async function aggregateNewContractsCountByDate() {
       })
     );
     console.log("NEW_CONTRACTS_COUNT_AGGREGATED_BY_DATE updated.");
+
+    // unique contract part
```

Comment:
Please, separate it into a dedicated function. Also, please take my above note into account for the new function name.

Reply:
rename it but it cannot be separate into a dedicated function since I need the array of date from new contracts count by date